server/lib/nl/fulfillment/triple.py

```diff
@@ -33,22 +33,36 @@
 _OUT_TITLE = 'The {property} for {entity} is:'
 _IN_TITLE = '{entity} is the {property} for:'
 _MULTI_ENTITY_TITLE = 'The {property} for {entity} are as follows:'
-_MAX_ENTITIES_IN_TITLE = 2
+_MAX_ENTITIES_IN_TITLE = 3
+
+
+# Gets the name for an entity
+def _entity_name(entity: Entity) -> str:
+  return entity.name or entity.dcid
 
 
 # Gets a title string for a list of entities
 def _get_entity_string(entities: List[Entity]) -> str:
   if len(entities) == 1:
-    return entities[0].name or entities[0].dcid
-  entities_cutoff_idx = min(_MAX_ENTITIES_IN_TITLE, len(entities) - 1)
-  entity_str = ', '.join(
-      [e.name or e.dcid for e in entities[:entities_cutoff_idx]])
-  entity_str += f' and {entities[entities_cutoff_idx].name or entities[entities_cutoff_idx].dcid}'
+    return _entity_name(entities[0])
+  entity_str = ''
+  max_entries = min(_MAX_ENTITIES_IN_TITLE, len(entities))
+  # Join together the first max_entries - 1 number of entities with a ', '
+  entity_str = ', '.join([_entity_name(e) for e in entities[:max_entries - 1]])
+  if max_entries < len(entities):
+    # If not all entities are named in the title, get the number of unnamed
```

Comment:
In this if case, we will only print 2 and that's intentional?

Reply:
yes, that's intentional! From the mocks, we want 'The [property] for [entity1], [entity2] and [N more other] are as follows'